js/server/assets/swagger/index.html

```diff
@@ -1,132 +1,111 @@
+        overflow: -moz-scrollbars-vertical;
+        overflow-y: scroll;
+      }
 
-  </defs>
-</svg>
+      *,
+      *:before,
+      *:after
+      {
+        box-sizing: inherit;
+      }
 
-<div id="swagger-ui"></div>
+      /* #region ArangoDB-specific changes */
+      body {
+        margin: 0;
+        background: #fff;
+        font-family: 'Open Sans', sans-serif;
+      }
 
-<script src="./swagger-ui-bundle.js"> </script>
-<script src="./swagger-ui-standalone-preset.js"> </script>
-<script>
-window.onload = function() {
-  var url = window.location.href.replace("index.html", "swagger.json").replace(/#.*/, "");
-  var jwt = sessionStorage.getItem("jwt");
-  
-  // Build a system
-  const ui = SwaggerUIBundle({
-    url: url,
-    dom_id: '#swagger-ui',
-    deepLinking: true,
-    tagsSorter: "alpha",
-    validatorUrl: null,
-    docExpansion: "list",
-    presets: [
-      SwaggerUIBundle.presets.apis,
-      SwaggerUIStandalonePreset
-    ],
-    plugins: [
-      SwaggerUIBundle.plugins.DownloadUrl
-    ],
-    layout: "StandaloneLayout",
-    requestInterceptor:
-      function (request) {
-        request.headers.Authorization = 'bearer ' + jwt;
-        return request;
+      .swagger-ui .scheme-container {
+        display: none;
+      }
+      .swagger-ui .info {
+        margin: 15px 0;
+      }
+      .swagger-ui a.nostyle span {
+        font-family: 'Open Sans', sans-serif;
+        font-weight: 400;
+      }
+      .swagger-ui code {
+        font-weight: 100 !important;
       }
-  })
 
-  window.ui = ui
-}
-</script>
-</body>
+      /* This fixes what seems to be a bug in the swagger-ui stylesheet
+       * causing syntax-highlighted code to have random white-on-gray text.
+       */
+      .swagger-ui .opblock-body pre span{
+        color: #000 !important;
+      }
 
+      /* The topbar lets the user pick arbitrary URLs.
+       * We don't need this feature in ArangoDB.
+       */
+      .topbar {
+        display: none;
+      }
+      /* #endregion */
+    </style>
+  </head>
+
+  <body>
+    <div id="swagger-ui"></div>
+
+    <script src="./swagger-ui-bundle.js"> </script>
+    <script src="./swagger-ui-standalone-preset.js"> </script>
+    <script>
+    window.onload = function() {
+      // Begin Swagger UI call region
+      const ui = SwaggerUIBundle({
+        //#region ArangoDB-specific changes
+        // Infer location of swagger.json from `location.href` instead of
+        // defaulting to the pet store example.
+        url: location.href.replace("index.html", "swagger.json").replace(/#.*/, ""),
+        //#endregion
+        dom_id: '#swagger-ui',
+        deepLinking: true,
+        presets: [
+          SwaggerUIBundle.presets.apis,
+          SwaggerUIStandalonePreset
+        ],
+        plugins: [
+          SwaggerUIBundle.plugins.DownloadUrl
+        ],
+        layout: "StandaloneLayout",
+        //#region ArangoDB-specific changes
+        validatorUrl: null,
+        docExpansion: "none",
```

Comment:
I'd consider this one a regression. Can you please change this to `"list"` to match the existing 3.4 behavior?

Reply:
Restored to "list"